pkg/server/options/virtual.go

```diff
@@ -17,9 +17,8 @@ limitations under the License.
 package options
 
 import (
-	"github.com/spf13/pflag"
-
 	virtualworkspacesoptions "github.com/kcp-dev/kcp/pkg/virtual/options"
+	"github.com/spf13/pflag"
```

Comment:
You want https://github.com/openshift-eng/openshift-goimports.

Reply:
Hmm, this was installed into my hack/tools directory and I had run `make imports`. Running it again doesn't result in a diff. I'll look into that